app/scripts/filters/resources.js

```diff
@@ -1022,6 +1022,20 @@ angular.module('openshiftConsole')
       }
     };
   })
+  .filter('humanizeTLSTermination', function () {
+    return function(string) {
+      switch (string) {
+        case 'edge':
+          return 'Edge';
+        case 'passthrough':
+          return 'Passthrough';
+        case 'reencrypt':
+          return 'Re-encrypt';
+        default:
+          return null;
```

Comment:
Fall back to returning the raw value:

```js
return string;
```

That's consistent with our other humanize filters and lets us use the filter without ever having to check for a null value.